apps/workers/utils.ts

```diff
@@ -14,3 +16,35 @@ export function withTimeout<T, Ret>(
     ]);
   };
 }
+
+export async function readPDFText(buffer: Buffer): Promise<{
+  text: string;
+  metadata: Record<string, string | Record<string, string>>;
```

Comment:
It seems that `Meta` is `Record<string, any>` according to https://github.com/modesty/pdf2json/blob/master/pdfparser.d.ts#L26 ?